arkouda/dataframe.py

```diff
@@ -2263,7 +2263,7 @@ def _parse_col_name(entryName, dfName):
     @staticmethod
     def from_return_msg(repMsg):
         """
-        Creates and returns a DataFrame based on return components from ak.util.attach
+        Creates and returns a DataFrame based on return components from ak.server_util.attach
```

Comment:
This is referring to `arkouda/util.py` and should not be changed.

Reply:
Fixed